lib/progressTracker.js

```diff
@@ -87,9 +87,7 @@ function track (instance, opts) {
         paddingRight: 1
       },
       drawHorizontalLine: () => false
-    })
-
-    logToStream(out)
+    }))
 
     if (opts.renderLatencyTable) {
```

Comment:
This will need an update as well, in conjunction with https://github.com/thekemkid/hdr-histogram-percentiles-obj/pull/2 I get:

```
$ node autocannon.js -c 100 -d 5 -l localhost:3000
Running 5s test @ http://localhost:3000
100 connections

Stat         2.5%    50%     97.5%   99%     Avg     Stdev   Max
Latency (ms) 3       4       8       11      4.66    2.19    93.68
Req/Sec      17503   19823   20623   20623   19384   1105.74 20619
Bytes/Sec    2883583 3.28 MB 3407871 3.41 MB 3.16 MB 196 kB  3.38 MB

Percentile      Latency (ms)
2.5
50              4
75              5
90              6
97.5
99              11
99.9
99.99
99.999

97k requests in 5s, 15.9 MB read
```